forge-gui/res/cardsfolder/upcoming/stangg_echo_warrior.txt

```diff
@@ -0,0 +1,14 @@
+Name:Stangg, Echo Warrior
+ManaCost:2 R G
+Types:Legendary Creature Human Warrior
+PT:3/4
+T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigStangg | TriggerDescription$ Whenever CARDNAME attacks, create Stangg Twin, a legendary 3/4 red and green Human Warrior creature token. It enters the battlefield tapped and attacking. For each Aura and Equipment attached to Stangg, create a token that's a copy of it attached to Stangg Twin. Sacrifice all tokens created this way at the beginning of the next end step.
+SVar:TrigStangg:DB$ Token | TokenScript$ stangg_twin | TokenOwner$ You | ImprintTokens$ True | RememberTokens$ True | TokenAttacking$ True | TokenTapped$ True | SubAbility$ CreateCopy
```

Comment:
I think you should add stangg_twin file to this PR as well

Reply:
Stangg twin is an existing token